Lib/fontmake/font_project.py

```diff
@@ -458,6 +479,8 @@ def save_otfs(
         )
 
         if interpolatable:
+            # XXX: Dead code?
```

Comment:
you're right. it's unused, since we now handle interpolatable outputs elsewhere (_build_interpolatable_masters). Please remove this and also the `interpolatable` keyword argument in `save_otfs` and all its references in the body of `save_otfs`